packages/babel-plugin-emotion/README.md

```diff
@@ -321,4 +321,4 @@ import css from '@emotion/css.macro'
 import styled from '@emotion/styled.macro'
 ```
 
-[create-react-app issue discussing macros](https://github.com/facebookincubator/create-react-app/issues/2730).
+Additionally [Create React App 2.0](https://reactjs.org/blog/2018/10/01/create-react-app-v2.html#whats-new) supports `babel-macros` out of the box.
```

Comment:
Could you remove this whole bit and add a bit to the above part about Create React App 2 supporting babel-plugin-macros